### DISCUSSION TITLE
How to get a Cryptomator recovery key?

### DISCUSSION THREAD

**Discussion by neorrk** [iterate-ch/cyberduck#14321] [Q&A]
Cyberduck/Mountain Duck Cryptomator vaults didn't provide me with a recovery key unlike the standalone Cryptomator app.

Is there any way to get the recovery key from the Duck made vaults?

**Comment by tatze96** [comment]
There isn't any way to get the recovery key from within Cyberduck or Mountain Duck, but you can display and save it from within the Cryptomator application.

I've added the steps to reach the recovery key below.
1. Mount the bookmark that includes the wanted vault through Mountain Duck.
2. Navigate into the wanted vault.
3. Open the `masterkey.cryptomator` file within the Cryptomator application to add the vault to the list.
4. Select the added vault in the Cryptomator application and click on the `Vault Options` button.
5. Open the _Password_ tab within the appearing options window and click on the `Display Recovery Key` button afterward.
6. Type your vault password in the appearing password prompt.
7. You should now see your recovery key.

You can only access the recovery key if you also use Mountain Duck unless the vault is saved on a server accessible through your local filesystem.
